### ISSUE TITLE
`dbt-osmosis==0.11.22` and `dbt-core==1.6.0` have incompatiable dependencies

### ISSUE THREAD

**Issue by waligob** [z3z1ma/dbt-osmosis#80]
Given the following dependencies in `pyproject.toml`:

```toml
[tool.poetry.dependencies]
python = ">3.9.7,<3.12"
dbt-bigquery = "1.6.1"
dbt-osmosis = {version = "0.11.22", extras = ["workbench"]}
```

`poetry lock` yields the following error:

```sh
$ poetry lock
Updating dependencies
Resolving dependencies... (5.8s)

Because dbt-bigquery (1.6.1) depends on dbt-core (>=1.6.0,<1.7.0)
 and no versions of dbt-core match >1.6.0,<1.7.0, dbt-bigquery (1.6.1) requires dbt-core (1.6.0).
And because dbt-core (1.6.0) depends on protobuf (>=4.0.0), dbt-bigquery (1.6.1) requires protobuf (>=4.0.0).
And because dbt-osmosis (0.11.22) depends on streamlit (1.20.0) which depends on protobuf (>=3.12,<4), dbt-bigquery (1.6.1) is incompatible with dbt-osmosis (0.11.22).
So, because inquirer-dbt depends on both dbt-bigquery (1.6.1) and dbt-osmosis (0.11.22), version solving failed.
```

**Comment by sdepablos** [comment]
Same issue. Streamlit is at 1.2.0, but it does not support Protobuf 4 until version 1.23.1 https://github.com/streamlit/streamlit/releases/tag/1.23.1

**Comment by z3z1ma** [comment]
Will take a look today

**Comment by z3z1ma** [comment]
Resolved in [0.12.0](https://github.com/z3z1ma/dbt-osmosis/releases/tag/v0.12.0) release, relaxed streamlit pin
